src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -112,6 +133,70 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
             return !Log.HasLoggedErrors;
         }
 
+        public async Task UploadAssets(ITaskItem item, SemaphoreSlim clientThrottle, bool allowOverwrite = false)
+        {
+            string relativeBlobPath = $"{feed.RelativePath}{item.GetMetadata("RelativeBlobPath").Replace("\\", "/")}"; 
+            if (string.IsNullOrEmpty(relativeBlobPath))
+                throw new Exception(string.Format("Metadata 'RelativeBlobPath' is missing for item '{0}'.", item.ItemSpec));
+
+            Log.LogMessage($"Uploading {relativeBlobPath}");
+
+            await clientThrottle.WaitAsync();
+
+            CloudTestTasks.AzureBlobLease blobLease = new CloudTestTasks.AzureBlobLease(feed.AccountName, feed.AccountKey, string.Empty, feed.ContainerName, relativeBlobPath, Log, "15", "5000");
```

Comment:
Could you please chop the args to new lines? Hard to read. (FWIW, I personally go for 100 columns as a soft limit.)